Core/GameEngine/Source/Common/Audio/AudioEventRTS.cpp

```diff
@@ -800,23 +800,22 @@ AsciiString AudioEventRTS::generateFilenameExtension( AudioType audioTypeToPlay
 //-------------------------------------------------------------------------------------------------
 void AudioEventRTS::adjustForLocalization(AsciiString &strToAdjust)
 {
-	const char *str = strToAdjust.reverseFind('\\');
-	if (!str) {
-		return;
+	const char *filename = strToAdjust.reverseFind('\\');
+	if (filename) {
+		filename += 1;
+	} else {
+		filename = strToAdjust.str();
```

Comment:
Instead of return, this would now work with audio files in the root directory. The game currently does not have any audio files in root, so this practically never hits.